release notes/v0.4.0.md

```diff
@@ -0,0 +1,117 @@
+xk6-browser v0.4.0 is here! :tada:
+
+This is a minor release with some bug fixes, a large portion of the `locator` feature implemented, improvements and code refactoring!
+
+
+## New features
+
+- Implement `page.locator(selector)` ([#310](https://github.com/grafana/xk6-browser/pull/310)).
```

Comment:
Can we use the past-participle as we used to do in previous release documents?

"Implement-ed" vs "Implement". Also, for the others.

We use the following formatting (notice the dot (not a biggie, but 😅):

```diff
- Implemented `X` ([#000]()).
+ Implemented `X`. ([#000]())
```